torch/_inductor/scheduler.py

```diff
@@ -1850,6 +1850,11 @@ def _init(self, nodes: List[ir.Operation]) -> None:
         if config.reorder_for_peak_memory:
             from .memory import reorder_for_peak_memory
 
+            self.nodes = comms.decide_global_ordering_of_comms(
+                self.nodes,
+                self.name_to_buf,
+                self.name_to_fused_node,
+            )
```

Comment:
I think we can just move the `self.nodes = comms.decide_global_ordering_of_comms(` call at line 1835 outside of its current `if config.reorder_for_compute_comm_overlap:` block